.github/workflows/main.yaml

```diff
@@ -59,19 +59,3 @@ jobs:
       track-name: 1.4
     secrets:
       CHARMCRAFT_AUTH: ${{ secrets.CHARMCRAFT_AUTH }}
-
-  publish-charm-on-push:
-    name: Publish Developer Charm To Branch
-    needs:
-      - lint-report
-      - static-analysis
-      - unit-tests-with-coverage
-      - integration-test
-    if: ${{ (github.ref_name != 'v1.4') && (github.event_name == 'push') }}
-    uses: canonical/sdcore-github-workflows/.github/workflows/publish-charm.yaml@v0.0.1
-    with:
-      branch-name: ${{ github.ref_name }}
-      charm-file-name: "sdcore-amf-k8s_ubuntu-22.04-amd64.charm"
-      track-name: 1.4
-    secrets:
-      CHARMCRAFT_AUTH: ${{ secrets.CHARMCRAFT_AUTH }}
```

Comment:
This does not seem to be broken, as the last merged PR published properly on the dev branch. It is useful in the testing the terraform modules for example.

Reply:
Actually I think publishing is completely broken on 1.4, not just from the dev branch

Reply:
It seems to have worked last week on 1.4

https://github.com/canonical/sdcore-amf-k8s-operator/actions/runs/10559094718/job/29251061822?pr=373

I would prefer to keep the feature as we are moving towards using Terraform for solutions deployment, and this will help us test through that exact method. I agree that we probably will not be using it very often on a release branch, but I'd prefer to have it for when we need it.

Reply:
Ok I found the issue: here's the fix on the sdcore-github workflow: 
- https://github.com/canonical/sdcore-github-workflows/pull/75

Reply:
Done